ui/src/timeMachine/constants/queryBuilder.ts

```diff
@@ -1,77 +1,82 @@
-import {WINDOW_PERIOD, OPTION_NAME} from 'src/variables/constants'
+export const AGG_WINDOW_AUTO = 'auto'
+export const AGG_WINDOW_NONE = 'none'
```

Comment:
is this being used anywhere? it seems like the only place it was being used is now commented out in line 59 of WindowSelector.tsx

Reply:
@ischolten it will be used in the next feature :)